releases/XCFramework/libzcashlc.xcframework/ios-arm64/libzcashlc.framework/Headers/zcashlc.h

```diff
@@ -1439,3 +1588,101 @@ struct FfiTxIds *zcashlc_create_proposed_transactions(const uint8_t *db_data,
+                                    const uint8_t *txid_bytes,
+                                    uintptr_t txid_bytes_len,
+                                    struct FfiTransactionStatus status);
+
+/**
+ * Frees an array of FfiTransactionDataRequest values as allocated by `zcashlc_transaction_data_requests`.
+ *
+ * # Safety
+ *
+ * - `ptr` if `ptr` is non-null it must point to a struct having the layout of [`FfiTransactionDataRequests`].
+ *   See the safety documentation of [`FfiTransactionDataRequests`].
+ */
+void zcashlc_free_transaction_data_requests(struct FfiTransactionDataRequests *ptr);
+
+/**
+ * Returns a list of transaction data requests that the network client should satisfy.
+ *
+ * # Safety
+ *
+ * - `db_data` must be non-null and valid for reads for `db_data_len` bytes, and it must have an
+ *   alignment of `1`. Its contents must be a string representing a valid system path in the
+ *   operating system's preferred representation.
+ * - The memory referenced by `db_data` must not be mutated for the duration of the function call.
+ * - The total size `db_data_len` must be no larger than `isize::MAX`. See the safety
+ *   documentation of pointer::offset.
+ * - Call [`zcashlc_free_transaction_data_requests`] to free the memory associated with the
+ *   returned pointer when done using it.
+ */
+struct FfiTransactionDataRequests *zcashlc_transaction_data_requests(const uint8_t *db_data,
+                                                                     uintptr_t db_data_len,
+                                                                     uint32_t network_id);
+
+/**
+ * Creates a Tor runtime.
+ *
+ * # Safety
+ *
+ * - `tor_dir` must be non-null and valid for reads for `tor_dir_len` bytes, and it must
+ *   have an alignment of `1`. Its contents must be a string representing a valid system
+ *   path in the operating system's preferred representation.
+ * - The memory referenced by `tor_dir` must not be mutated for the duration of the
+ *   function call.
+ * - The total size `tor_dir_len` must be no larger than `isize::MAX`. See the safety
+ *   documentation of pointer::offset.
+ * - Call [`zcashlc_free_tor_runtime`] to free the memory associated with the returned
+ *   pointer when done using it.
+ */
+struct TorRuntime *zcashlc_create_tor_runtime(const uint8_t *tor_dir, uintptr_t tor_dir_len);
+
+/**
+ * Frees a Tor runtime.
+ *
+ * # Safety
+ *
+ * - If `ptr` is non-null, it must point to a struct having the layout of [`TorRuntime`].
+ */
+void zcashlc_free_tor_runtime(struct TorRuntime *ptr);
+
+/**
+ * Fetches the current ZEC-USD exchange rate over Tor.
+ *
+ * The result is a `u128` containing a packed decimal:
+ * - Bits 16-23: Contains "e", a value between 0-28 that indicates the scale.
+ * - Bit 31: the sign of the Decimal value, 0 meaning positive and 1 meaning negative.
+ * - Bits 32-127: Contains the representation of the Decimal value as a 96-bit integer.
```

Comment:
```suggestion
 * - Bits 32-127: Contains the mantissa of the Decimal value as a 96-bit integer.
```

Reply:
These changes should be made in the Rust code; this is autogenerated.

Reply:
Would `significand` be better? https://en.wikipedia.org/wiki/Significand

Reply:
I don't actually know whether this is the mantissa (i.e. the scaled decimal fraction excluding an implicit 1 unit) or the significand. [Apple's documentation for NSDecimal](https://developer.apple.com/documentation/foundation/nsdecimalnumber/1416003-init) uses "mantissa" for the significand, but that's the API documentation, not the binary format documentation. I was unable to find the latter.